### PR TITLE
Added Auth Page

### PR DESCRIPTION
**Pull request by sanjitsarkar**
In this PR,
I have added authentication feature.
Preview Link: [Home Page]( https://zengram.netlify.app/)
**Login Page**
Screenshot from 2022-05-12 01-21-52
**Signup Page**
Screenshot from 2022-05-12 01-21-55

### CODE REVIEW

**Review by bossmonkey98 — APPROVED** [review]
Code looks clean good to merge

File: src/pages/AuthPage/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+      leading-tight
+      uppercase
+      rounded
+      shadow-md
+      hover:bg-primary hover:shadow-lg
+      focus:bg-primary focus:shadow-lg focus:outline-none focus:ring-0
+      active:bg-primary active:shadow-lg
+      transition
+      duration-150
+      ease-in-out"
```
Comment (iRohitGaur) [inline src/pages/AuthPage/LoginPage.jsx:133]: suggestion: too many classes. try to make it less modular

File: src/pages/AuthPage/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      leading-tight
+      uppercase
+      rounded
+      shadow-md
+      hover:bg-secondary hover:shadow-lg
+      focus:bg-secondary focus:shadow-lg focus:outline-none focus:ring-0
+      active:bg-secondary active:shadow-lg
+      transition
+      duration-150
+      ease-in-out"
```
Comment (iRohitGaur) [inline src/pages/AuthPage/LoginPage.jsx:160]: same here

**Review by iRohitGaur — APPROVED** [review]
Nice work! 

Gave some suggestions, please look into it.

Good to merge 👍🏻

File: src/pages/AuthPage/LoginPage.jsx (hunk 1 above)
Comment (sanjitsarkar) [reply src/pages/AuthPage/LoginPage.jsx:133]: Okay will make a separate component for input field and reuse it

File: src/pages/AuthPage/LoginPage.jsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        text-base
+        font-normal
+        text-gray-700
+        bg-white bg-clip-padding
+        border border-solid border-gray-300
+        rounded
+        transition
+        ease-in-out
+        m-0
+        focus:text-gray-700 focus:bg-white focus:border-primary focus:outline-none"
```
Comment (saurabhchirde) [inline src/pages/AuthPage/LoginPage.jsx:53]: try to use less class

**Review by saurabhchirde — APPROVED** [review]
I couldn't find any issue in your code, gave minor suggestions regarding the number of classes you have used. Otherwise regarding the UI, the background colour you have used is harsh to the eyes, try to make it subtle, else looks good to merge.

**Pull request merged**